test/Confluent.Kafka.IntegrationTests/TemporaryTopic.cs

```diff
@@ -23,19 +23,24 @@ namespace Confluent.Kafka
 {
     public class TemporaryTopic : IDisposable
     {
-        IAdminClient adminClient;
+        private string bootstrapServers;
+        
+        public string Name { get; set; }
 
         public TemporaryTopic(string bootstrapServers, int numPartitions)
```

Comment:
I think it might be better name prefix all these topics with a known name, librdkafka has rdktest_.... or something, which allows a tool to scan and delete these topics at any point.
The problem with the current approach is that temporary topics will not be cleaned up if the tests crash.

The way the librdkafka integration tests does this is use the prefix, and then after all tests have finished it scans through and deletes the topics (optional).

Reply:
yep, I can do that, makes it generally more robust. that said, my integration test clusters are always temporary, and the docker scripts i'm setting up to auto-run integration tests atm will also bring up new clusters for a given test run, so it doesn't really matter.